index.js

```diff
@@ -560,25 +561,25 @@ alexa.app = function(name) {
   // @throws Error when router or expressApp options are not specified
   // @returns this
   this.express = function(options) {
-    if (!options.expressApp) {
-      throw new Error("You must specify an express instance to attach to.");
+    if (!options.expressApp && !options.router) {
+      throw new Error("You must specify an express app or an express router to attach to.");
     }
 
-    if (!options.router) {
-      throw new Error("You must specify an express router to attach.");
+    if (options.expressApp && options.router) {
+      console.warn("Since alexa-app@4.0.0, you are no longer required to pass both 'expressApp' and 'router'.");
+      console.warn("When both passed, 'alexa-app' would be attached to 'expressApp' instance.");
```

Comment:
IMO, the warning message I would have done is something along the lines of "`'expressApp' and 'router' are both specified, using 'expressApp'...`". Generally, I want to give the user just the warning itself, and if they want to know more about it, they could look up [UPGRADING.md](https://github.com/alexa-js/alexa-app/blob/master/UPGRADING.md) for more information.